mmv1/third_party/terraform/services/resourcemanager/ephemeral_google_service_account_token_test.go

```diff
@@ -0,0 +1,141 @@
+package resourcemanager_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestEphemeralServiceAccountToken_basic(t *testing.T) {
+	t.Parallel()
+
+	serviceAccount := envvar.GetTestServiceAccountFromEnv(t)
+	targetServiceAccountEmail := acctest.BootstrapServiceAccount(t, "acctoken", serviceAccount)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() { acctest.AccTestPreCheck(t) },
+		ExternalProviders: map[string]resource.ExternalProvider{
+			"time": {},
+		},
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccEphemeralServiceAccountToken_basic(targetServiceAccountEmail, serviceAccount),
+			},
+		},
+	})
+}
+
+func TestEphemeralServiceAccountToken_withDelegates(t *testing.T) {
+	t.Parallel()
+
+	serviceAccount := envvar.GetTestServiceAccountFromEnv(t)
+	targetServiceAccountEmail := acctest.BootstrapServiceAccount(t, "acctoken-delegate", serviceAccount)
+	delegateServiceAccountEmail := acctest.BootstrapServiceAccount(t, "acctoken-delegate-sa", serviceAccount)
```

Comment:
```suggestion
	targetServiceAccountEmail := acctest.BootstrapServiceAccount(t, "targer", serviceAccount)
	delegateServiceAccountEmail := acctest.BootstrapServiceAccount(t, "delegate", serviceAccount)
```

We need to shorten these arguments to avoid errors like:

> Error 400: The account ID "tf-bootstrap-sa-acctoken-delegate" does not have a length between 6 and 30., badRequest